wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluators.scala

```diff
@@ -51,7 +51,7 @@ object EngineFunctionEvaluators {
       EvaluatedValue(WomSingleFile(ioFunctionSet.pathFunctions.stderr), Seq.empty).validNel
   }
 
-  private val ReadWaitTimeout = 60.seconds
+  private val ReadWaitTimeout = 57.seconds
```

Comment:
Not an accidental check-in, as it was genuinely a very useful to know which timeout was firing; but not strongly attached to this.

Reply:
I lean toward leaving this at 60s, since that feels more conservative and less confusing for users, but also don't feel too strongly.